docs/tutorials/tabular_prediction/tabular-multimodal-text-others.md

```diff
@@ -1,8 +1,8 @@
-# TabularPredictor for Multimodal Data Tables: Combining BERT and Classical Tabular Models
+# Multimodal Data Tables: Combining BERT/Transformers and Classical Tabular Models
 
 :label:`sec_tabularprediction_text_multimodal`
 
-We will introduce how to use AutoGluon Tabular to deal with multimodal tabular data that involves text, numeric, and categorical features. This type of data is prevalent in real world applications. In AutoGluon, **raw text data** is considered as a first-class citizen of data tables. AutoGluon Tabular can help you train and combine a diverse set of models including the classical ones like LightGBM/CatBoost and the more recent Pretrained Language Model (PLM) based multimodal network that is introduced in Section ":ref:`sec_textprediction_architecture`" of :ref:`sec_textprediction_multimodal`.
+Here we introduce how to use AutoGluon Tabular to deal with multimodal tabular data that contains text, numeric, and categorical columns. In AutoGluon, **raw text data** is considered as a first-class citizen of data tables. AutoGluon Tabular can help you train and combine a diverse set of models including classical tabular models like LightGBM/RF/CatBoost as well as our pretrained MLP model based multimodal network that is introduced in Section ":ref:`sec_textprediction_architecture`" of :ref:`sec_textprediction_multimodal` (used by AutoGluon's `TextPredictor`).
```

Comment:
"as well as our pretrained MLP model based" --> "as well as our Pretrained Language Model (PLM) based".

Reply:
sorry was supposed to say NLP, fixed now.
I thought we are not calling these language models (and abbreviation PLM is not used anywhere)?

Reply:
pretrained NLP looks good.